string sensor_name
---
string message
